cli/command/load_balancer.go

```diff
@@ -62,7 +62,12 @@ func (l *LoadBalancerCommand) Command() cli.Command {
 					cli.StringFlag{
 						Name:  "healthcheck-target",
 						Value: dhc.Target,
-						Usage: "Health check target in format 'PROTOCOL:PORT' or 'PROTOCOL:PORT/WITH/PATH'",
+						Usage: "Health check target in format 'PROTOCOL:PORT' or 'PROTOCOL:PORT/WITH/PATH', used for load balancer type 'ELB'",
```

Comment:
I believe you intended to say CLB instead of ELB here.

Reply:
✅